examples/Bremsstrahlung/include/simulation_defines/param/densityConfig.param

```diff
@@ -0,0 +1,99 @@
+/**
+ * Copyright 2013-2017 Axel Huebl, Heiko Burau, Rene Widera, Felix Schmitt,
+ *                     Richard Pausch
+ *
+ * This file is part of PIConGPU.
+ *
+ * PIConGPU is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * PIConGPU is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with PIConGPU.
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+
+
+#pragma once
+
+#include "particles/densityProfiles/profiles.def"
+/* preprocessor struct generator */
+#include "preprocessor/struct.hpp"
+
+namespace picongpu
+{
+namespace SI
+{
+    /** The maximum density in particles per m^3 in the gas distribution
+     *  unit: ELEMENTS/m^3
+     *
+     * He (2e- / Atom ) with 1.e15 He / m^3
+     *                      = 2.e15 e- / m^3 */
+
+    constexpr float_64 BASE_DENSITY_SI = 5.9e28; // solid gold
+
+}
+
+//##########################################################################
+//############## special density profiles ##################################
```

Comment:
this block is removed in new files, see `src/picongpu/include/simulation_defines/param/densityConfig.param`